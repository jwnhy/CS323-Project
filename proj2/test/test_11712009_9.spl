struct a {
    int b;
};
int main () {
    return a.c;
}