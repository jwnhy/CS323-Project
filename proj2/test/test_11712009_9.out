Error type 1 at Line 5: Variable not found in scope.
Error type 13 at Line 5: Not Structure.
Error type 8 at Line 5: Invalid return type.
